hub/core/dataset/dataset.py

```diff
@@ -476,7 +476,7 @@ def pytorch(
         collate_fn: Optional[Callable] = None,
         pin_memory: bool = False,
         shuffle: bool = False,
-        buffer_size: int = 512,
+        buffer_size: int = 2048,
```

Comment:
The shuffle `buffer_size` parameter is repeated in two source files, and am assuming these must always be the same. If that is the case, going forward there should be a single configuration source that specs various parameters that are system wide.

Reply:
Valid point.